src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -46,16 +50,24 @@
   </ItemGroup>
   <!-- Carry a copy of MathF where not available -->
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)' == 'true' OR $(TargetGroup.StartsWith('netstandard'))">
-    <Compile Include="..\..\Common\src\System\MathF.netstandard.cs">
+    <Compile Include="$(CommonPath)\System\MathF.netstandard.cs">
       <Link>System\MathF.netstandard.cs</Link>
     </Compile>
   </ItemGroup>
+  <!-- Optimize Quaternion as Vector4 for netcoreapp -->
+  <!-- Jit issue for other runtimes https://github.com/dotnet/coreclr/issues/15237 -->
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND $(TargetGroup.StartsWith('netcoreapp2'))">
```

Comment:
CoreLib reference can be used for `netcoreapp` only.

Reply:
Always gives me this when using netcoreapp
```
C:\GitHub\corefx\buildvertical.targets(168,5): error : 
Could not find a configuration for ProjectReference
'C:\GitHub\corefx\\external\runtime\runtime.depproj' from configurations
 netcoreapp-Windows_NT;
 netcoreapp-Unix;
 netcoreapp2.0-Windows_NT;
 netcoreapp2.0-Unix;
 uap;
 uapaot; 
 mono 
when building 'System.Numerics.Vectors' for configuration 
 netcoreapp
[C:\GitHub\corefx\src\System.Numerics.Vectors\src\System.Numerics.Vectors.csproj]
```